tests/test_toolchain.py

```diff
@@ -0,0 +1,62 @@
+import sys
+import pytest
+import mock
+from pythonforandroid.toolchain import ToolchainCL
+from pythonforandroid.util import BuildInterruptingException
+
+
+class TestToolchainCL:
+
+    def test_help(self):
+        """
+        Calling with `--help` should print help and exit 0.
+        """
+        argv = ['toolchain.py', '--help', '--storage-dir=/tmp']
+        with mock.patch('sys.argv', argv), pytest.raises(SystemExit) as ex_info, \
```

Comment:
could we avoid this backslash?
I suggest something like:
```
        with mock.patch('sys.argv', argv), pytest.raises(
            SystemExit
        ) as ex_info, mock.patch(
            'argparse.ArgumentParser.print_help'
        ) as m_print_help:
            ToolchainCL()
```
Which also will be below 80 characters, so it will be pep8 friendly

Reply:
Addressed